src/ResourceManager/Sql/Commands.Sql/help/Add-AzureRmSqlElasticJobStep.md

```diff
@@ -0,0 +1,510 @@
+---
+external help file: Microsoft.Azure.Commands.Sql.dll-Help.xml
```

Comment:
Few comment on help files, but I'll review more closely the second time:

Make sure you regenerate so you get all the changes to the cmdlet above.
Make sure you change the output type to be singular (this might not get automatically done)
Make sure you have example output for each file (looks like this might already be done) and that every parameter has a description (also might be already done, but would be good to double check).